packages/compass-crud/src/stores/crud-store.js

```diff
@@ -378,17 +353,9 @@ const configureStore = (options = {}) => {
      */
     copyToClipboard(doc) {
       track('Document Copied', { mode: this.modeForTelemetry() });
-      const documentJSON = doc.toEJSON();
-      let input = document.createElement(INPUT);
-      input.type = TYPE;
-      input.setAttribute(STYLES, DISPLAY);
-      input.value = documentJSON;
-      document.body.appendChild(input);
-      input.select();
-      const success = document.execCommand(COPY);
-      document.body.removeChild(input);
-      input = null;
-      return success;
+      const documentEJSON = doc.toEJSON();
+      // eslint-disable-next-line no-undef
+      void navigator.clipboard.writeText(documentEJSON);
```

Comment:
Updated the copy pasting, also added 2 tests https://github.com/mongodb-js/compass/pull/3118/files#diff-4ac7fe029a57f95f04bb2b93dbcfc6a83fe0fcbf2b46b89103577efbe3329199R213